### PR TITLE
adding keys for shivanshu21

### PR DESCRIPTION
**Pull request by shivanshu21**
Added my public key to puppet-rjil code

### CODE REVIEW

File: hiera/data/secrets/common.yaml [hunk 1]
```diff
@@ -42,6 +42,8 @@ rjil::system::accounts::local_users:
...
   bhavin: {realname: 'Bhavin Ajmera', sshkeys: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCpjuYSg51zZvf7Yz+W33TyZPWMN90cZm+Z0xDXN0SkvPZ/7O2CUgbV9NWFGYjZ1HEhElZoUDMbYsskXuyL2LxzveHEswUHRpxI5J+kiONDrH1uSrMElbMD0jgi/awDHilMrtb+qehiuzi+iOY+mZP61hRk9+51NdtdLpwO6NAgqa6UrMF0jINs1jmAhI7ByJjjGHtC+LnukEC4ASpzFSv8KdcDqIwjwfEhqd+lcurtscqekwqt0ZCCysM7VB6zm5NVRsAsm3ya4SBv5xiaJX0ucOXNrzp+aWz+qpBLLTzmFyrtMTvSUIHheIe2uvews865aXr0CiiVTQnFbQ0UtnWJ', password: '*'}
+  shivanshu21: {realname: 'Shivanshu Goswami', sshkeys: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDW+laRtbIKWWSZ9Ro/RPW6odtWpqhUjonAyLj7CI9cYDMWKF1OwvAt7kRuPWtvTtBIYtZbIsDE3tryw46+oVO39B0UWJW+Q5e7iqfjljsJOAi3ORTSRH6Eo2+ejSQMsYxVliQK2krvLeCQWPss8sGf9dj8IBuJi6HwOydql8CFPRXnfOygK2EWdhFxihjm2p4iQatVV/byqBqhhqHLsqIn4VOsLIEkWZtDR43V/jOvi+Hbh+dCB4Do9YvSNH2kyTDPB5z/9GTQ7uclwtiw7V5d6esrAswJozJi0FcAPvsax86GG93rGn+LhvgH8921PCS1+IFiiODVl7LRcbMkHjCH shivanshu@Shivanshu', password: '*'}
+
```
Comment (jiocloud-tools) [inline hiera/data/secrets/common.yaml]: Please remove this empty line below your SSH key. Then it's good to merge.

Comment (mayankkapoor) [inline hiera/data/secrets/common.yaml]: Please remove this empty line below your SSH key. Then it's good to merge.

**Comment by anshprat** [comment]
Please squash the commits into one.

**Comment by mayankkapoor** [comment]
@shivanshu21 This needs a rebase

**Comment by anshprat** [comment]
merged as 850, closing this now

**Pull request closed without merging**